scripts/ccpp_capgen.py

```diff
@@ -526,7 +538,7 @@ def parse_scheme_files(scheme_filenames, run_env):
         # parse metadata file
         mtables = parse_metadata_file(filename, known_ddts, run_env)
         fort_file = find_associated_fortran_file(filename)
-        ftables = parse_fortran_file(fort_file, run_env)
+        ftables, additional_routines = parse_fortran_file(fort_file, run_env)
```

Comment:
A good suggestion! I hadn't thought about it but have now looked into it and it would be a bit of a lift to add to ftables due to the way parsing works...